azure-pipelines.yml

```diff
@@ -232,9 +246,4 @@ jobs:
       - task: CmdLine@2
         displayName: 'Run Unit Tests'
         inputs:
-          script: dotnet test $(PathToCommunityToolkitUnitTestCsproj) -c Release
-
-      - task: CmdLine@2
-        displayName: 'Pack CommunityToolkit NuGets'
-        inputs:
-          script: 'dotnet pack -c Release $(PathToCommunityToolkitCsproj) -p:PackageVersion=$(NugetPackageVersion) --output $(Build.ArtifactStagingDirectory)/nuget -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg'
```

Comment:
Removed `dotnet pack` here because we don't use the NuGet Package generated by this step and Azure DevOps kept throwing an error:
> ##[error]Bash exited with code '137'.

[Exit Code 137](https://www.analyticsinsight.net/open-source-help-what-is-exit-code-137-and-can-you-fix-it/) indicates the Kubernetes container ran out of memory. I assume this is an Azure DevOps bug.

https://dev.azure.com/dotnet/CommunityToolkit/_build/results?buildId=77448&view=logs&j=c44e3ce6-80b8-5090-9435-9627042472dc&t=268a63b1-8775-5bcc-b710-4db0706d42a0&l=14